### DISCUSSION TITLE
protein vs gene id for less well-annotated species

### DISCUSSION THREAD

**Discussion by silasmellor** [vdemichev/DiaNN#295] [Q&A]
Hey there, first of all thanks for making this super useful tool available to the community and for active support! 

I am analysing DIA data from Petunia species with recently published and unfortunately poorly annotated genomes (i previously wrote about issues with FASTA digestion for library-free quantification, which may be caused by this as well). I have generated a spectral library based on DDA data using fragpipe and used that to ID proteins in the DIA data with DIA-NN. I notice that in the main report after processing the files, the gene column is left blank (presumably because the FASTA headers don't contain the usual uniprot GN annotation), and so is the genes.MaxLFQ.unique (and other related gene quantification columns). Is it OK in this case to simply use the PG.MaxLFQ values for downstream analysis (or the protein.group LFQ values you provide an example for in the diannR package documentation), or would you take a different approach to infer the corresponding unique gene LFQ values? I should mention the PG column only contain single protein entries in the main report.

An additional question that is a bit unclear to me, what is the difference between the PG.MaxLFQ column values in the main report, and the values obtained by using the diannR command for protein groups?
`protein.groups <- diann_maxlfq(df[df$Q.Value <= 0.01 & df$PG.Q.Value <= 0.01,], group.header="Protein.Group", id.header = "Precursor.Id", quantity.header = "Precursor.Normalised")`

Thanks,
Silas

**Comment by vdemichev** [comment]
Hi Silas,

I would suggest to disable Protien inference in DIA-NN and leave the FASTA database field empty. Then it will just use protein groups generated by FragPipe.

Yes, PG.MaxLFQ is good. diann R package has a slightly different algorithm. Also, filtering applied by DIA-NN itself might be different.

Best,
Vadim

**Reply by silasmellor** [reply]
Great, i will carry on with the PG.MaxLFQ. Thanks for the quick response!
Best,
Silas
